pom.xml

```diff
@@ -3115,6 +3115,9 @@
                 <param>
                   com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocVariableCheckTest
                 </param>
+                <param>
+                  com.puppycrawl.tools.checkstyle.checks.coding.MultipleStringLiteralsCheckTest
+                </param>
```

Comment:
Needed to add this here so that we have coverage for `stripIndentAndInitialNewLineFromTextBlock()` . I didn't want to add artificial test input in `CheckUtilTest.java`,  since we should be testing on real code.